spec/foodzone/venues/finder_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../../../app/foodzone/venues/finder'
+
+describe Venues::Finder do
+  let(:finder) { described_class }
+  let(:venue_repository) { double :venue_repository }
+
+  describe '::find' do
+    context 'when passing a name' do
+      let(:venue_name) { 'my_venue' }
+      it 'returns the venue by name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.